src/app/dim-ui/CheckButton.m.scss

```diff
@@ -15,6 +15,7 @@
   }
   > *:last-child {
     margin-left: 6px;
+    flex-shrink: 0;
```

Comment:
Just something I found while I was playing with checkbox versions. This will ensure the switch component never gets squeezed to be smaller.

Reply:
I noticed you made some changes to this the other day @bhollis. Should I leave these or remove it?

Reply:
This shouldn't conflict